src/Components/Bookpage/BookDetails/index.js

```diff
@@ -12,7 +17,14 @@ function BookDetails({ image, title, author, year, pageCount, genre, blurb }) {
                 <p>{year}</p>
                 <p>{pageCount} pages</p>
                 <p>Genre: {genre.name}</p>
+                <p>
+                    <a href="#reviews">{reviews.length} reviews</a> - {avgScore.toFixed(1)} / 5 stars
```

Comment:
Check with Ash if using an anchor tag here is a good idea as it may re-render/refresh. I don't know the answer to this question...